bootstrap/010_helpers.sql

```diff
@@ -149,3 +149,44 @@ $$
 
   return str;
 $$;
+
+create or replace view internal.regions as
```

Comment:
do we not have access to `show regions`?

Reply:
Not sure off the top of my head. I remembered we had a similar enumeration in the streamlit Python code and lifted it here.

Reply:
yeah, I think we were being lazy in teh streamlilt code. Wondering if we really need to be

Reply:
Pushed a view, built from `show regions`, into the application package which appears to do the trick.